Symfony/CS/Fixer/Contrib/MethodArgumentDefaultValueFixer.php

```diff
@@ -0,0 +1,203 @@
+            $nextRelevantTokenContent = $tokens[$tokens->getNextMeaningfulToken($nextRelevantTokenIndex)]->getContent();
+
+            if (in_array($nextRelevantTokenContent, $this->argumentTerminatorTokens, true)) {
+                return true;
+            }
+
+            $nextRelevantTokenIndex = $this->findNextVariableOrTokenOfKind($tokens, $nextRelevantTokenIndex, $this->functionDefinitionTerminatorTokens);
+        }
+
+        return false;
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     * @param array  $relevantTokens
+     *
+     * @return int|null
+     */
+    private function findNextVariableOrTokenOfKind(Tokens $tokens, $index, array $relevantTokens)
+    {
+        $variableIndices = array_keys($tokens->findGivenKind(T_VARIABLE));
+
+        $nextVariableIndex = $this->getFirstValueBiggerThan($variableIndices, $index);
+        $nextRelevantTokenIndex = $tokens->getNextTokenOfKind($index, $relevantTokens);
+
+        if (null === $nextVariableIndex) {
+            return $nextRelevantTokenIndex;
+        }
+
+        if (null === $nextRelevantTokenIndex) {
+            return $nextVariableIndex;
+        }
+
+        return ($nextVariableIndex < $nextRelevantTokenIndex) ? $nextVariableIndex : $nextRelevantTokenIndex;
+    }
+
+    /**
+     * @param array $values
+     * @param int   $minimumValue
+     *
+     * @return int|null
+     */
+    private function getFirstValueBiggerThan(array $values, $minimumValue)
+    {
+        foreach ($values as $value) {
+            if ($value > $minimumValue) {
+                return $value;
+            }
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return bool
+     */
+    private function hasDefaultValueAfterIndex(Tokens $tokens, $index)
+    {
+        $nextTokenIndex = $tokens->getNextTokenOfKind($index, $this->defaultValueTokens);
+        $nextToken = $tokens[$nextTokenIndex];
+
+        return $nextToken->getContent() === '=';
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return bool
+     */
+    private function isDefaultArgumentAfterIndex(Tokens $tokens, $index)
+    {
+        $nextTokenIndex = $tokens->getNextTokenOfKind($index, $this->immediateDefaultValueTokens);
+        $nextToken = $tokens[$nextTokenIndex];
+
+        return $nextToken->getContent() === '=';
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $nextVariableIndex
+     */
+    private function removeDefaultArgument(Tokens $tokens, $nextVariableIndex)
+    {
+        $currentIndex = $nextVariableIndex;
+
+        while (!in_array($tokens[$currentIndex + 1]->getContent(), $this->argumentTerminatorTokens, true)) {
+            $tokens[++$currentIndex]->clear();
+        }
+    }
+
+    /**
+     * @return int
+     */
+    public function getPriority()
+    {
+        return -49;
```

Comment:
you should think if the fixer you are creating need to run after or before any other existing fixer ?